plasmapy/physics/tests/test_parameters.py

```diff
@@ -552,17 +546,17 @@ def test_gyroradius():
     assert gyroradius(B2, Vperp=Vperp2, particle='alpha') == \
```

Comment:
kill all the continuation slashes :see_no_evil:

Reply:
This file, at least, is clean.